gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobTask.java

```diff
@@ -211,14 +211,15 @@ public TaskResult run() {
       try {
         this.jobsMapping.deleteMapping(jobUri);
       } catch (Exception e) {
+        log.warn("Failed to delete jobs mapping for job: {}", jobUri, e);
         return new TaskResult(TaskResult.Status.FAILED,"Cannot delete jobs mapping for job : " + jobUri);
       }
     }
   }
 
   @Override
   public void cancel() {
-    log.info("Cancelling planning job {}", this.planningJobId);
+    log.info("Cancelling planning job {} | jobTaskMetrics: {}", this.planningJobId, this.jobTaskMetrics);
```

Comment:
If you're logging the metrics like this it would just print a reference to the java object? Is that intended?

Reply:
Thanks for catching this, fixed.